lib/themes/dosomething/paraneue_dosomething/templates/reportback/reportback-permalink.tpl.php

```diff
@@ -74,8 +74,10 @@
           <?php else: ?>
             <div class="cta">
               <div class="wrapper">
-                <?php if ($node->secondary_call_to_action): ?>
+                <?php if ($node->secondary_call_to_action && $node->status == 'active'): ?>
                   <p class="cta__message"><?php print $node->secondary_call_to_action; ?></p>
+                <?php elseif($node->status == 'closed'): ?>
```

Comment:
the OCD in me can't help but mention there should be a space between the `elseif` and condition :mask: